core/pom.xml

```diff
@@ -54,6 +54,10 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jdk8</artifactId>
```

Comment:
This dependency is already included in the dist. @vmuzikar Is it something we want to remove as part of the JDK 21 upgrade?

Reply:
Yeah we're already including it in the dist, so I figured we could re-use it here. Note that when Jackson 3.0 is released this functionality will be included as it is [Java 8+ compatible out](https://github.com/FasterXML/jackson-modules-java8?tab=readme-ov-file#jackson-30) of the box, removing the need to carry this dependency in the future. The release is slated for some time [early this year](https://github.com/FasterXML/jackson/wiki/Jackson-Releases).

Reply:
We're not removing it as part of the JDK 21 (@mabartos can correct me). Despite the intention of keeping the `core` deps at minimum, I see not much harm in adding this one. And it [does not have any additional dependencies](https://central.sonatype.com/artifact/com.fasterxml.jackson.datatype/jackson-datatype-jdk8/dependencies).